src/features/all-drops/components/AllDrops.tsx

```diff
@@ -146,37 +134,30 @@ export default function AllDrops() {
 
     let nftHref = '';
```

Comment:
```suggestion
    let nftHref;
```

would suggest to leave it undefined then in **line 154,** you don't need to check if its NFT
```
media: nftHref,